Kata_v1.0.0/src/kata_v2.js

```diff
@@ -58,10 +58,21 @@ const reverseArray = (array) => {
   return array.reverse();
 };
 
+const boxSequence = (step) => {
+	var boxes;
+	if(step % 2 !==0 ) {
+		return boxes = step + 2;
+	}else if (step % 2 === 0 ){
+		return boxes = step;
+	}
+	return boxes;
```

Comment:
you can convert this into a single line using ternary operator

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator resource